introduction/faq.md

```diff
@@ -610,13 +603,14 @@ Qubes is slower when reading from the disk because of the VM overhead, which is
-Please don't ask for your favorite program to be installed by default or for some setting that obviously varies by user preference to be changed so that it matches *your* preference.
-This is an incredibly selfish attitude that demonstrates a complete lack of consideration for the thousands of other Qubes users who don't happen to share your preferences.
+When responding to one of our surveys or otherwise participating in structured user research, this is exactly the sort of feedback we love to hear. In the context of structured user research, this kind of feedback has a place and can be put to good use. On email lists or as discussion threads in user forums, the core team is unable to productively engage in these sorts of requests. Even if a consenssus forms in a social channel—how does that group of users measure-up to the needs of the other [**+40,000 current users of Qubes OS**](https://www.qubes-os.org/statistics/)? 
+
+Many, many, many things factor into how the core team prioritizes new features, and changes to existing features—not the least of which, are urgent security matters, things breaking, or things that have been on our minds and in active discussion for many months—if not years. We'd love to move faster, but right now that is the reality of the project's few staff resources and many volunteers, all working very, very hard. To keep ourselves sane and our GitHub manageable, GitHub Issues filed with these asks are subject to immediate closure as `wontfix`—and we ask folks to please just not create those.
+
+In 2019 we began our first structured research effort to learn more about [who's using Qubes](https://www.qubes-os.org/news/2020/11/26/qubes-survey-results/). Between that survey and follow-up interviews, we've learned that some of our users are security researchers in academia, while others are human rights workers still struggling to get their heads around using GPG; many of our users are developers, while many others are less-technically adept but still die-hard Linux and OSS enthusiasts. Many others are workers in government or enterprise environments. Some of our users can't imagine using Qubes with anything other than an i3 DE with D-Menu, while others still struggle with Qubes leaving KDE for XFCE as our default DE. 
+
+TL;DR, chances are that if thousands of users will truly benefit from something, that there's already a GitHub issue for it. That issue could be many months (or years) old, too. Or, there's a non-negotiable reason something is not the default already. We love our user community. We're just a very small, modest team, and ask for your humble understanding in not overwhelming us with what we'll never be able to meaningfully act upon. 
```

Comment:
This is a bit too rambling and hand-wringing for my taste. It sounds too apologetic about things that merit no apology and beats around the bush instead of giving a straight answer.

Reply:
Perhaps we could discuss some more, perhaps offline? 

The way the current answer is written, assumes the end user has some (any) visibility into how software is made. From my experiences interviewing users for user research, most have none. Or, their assumptions are misaligned with how big a team is, how many users there really are, or what the diversity of the user base is. So, it's not being apologetic at all; it's providing information to give the user an opportunity to see why we may think their request is selfish. 

Especially as more and more non-technical and not-FOSS-community users come into the Qubes fold, I'd like to see more of the FAQ and intro content assume less about a user's knowledge of how our sausage gets made. If that makes sense.